### PR TITLE
Fix setLocale does not work when Locale enum is from two packages

### PR DESCRIPTION
**Pull request by fzyzcjy**
Close https://github.com/Tienisto/slang/issues/137

### CODE REVIEW

File: slang_flutter/lib/slang_flutter.dart [hunk 1]
```diff
@@ -158,8 +157,7 @@ class _TranslationProviderState<E extends BaseAppLocale<E, T>,
   }
 }
 
-class InheritedLocaleData<E extends BaseAppLocale<E, T>,
-    T extends BaseTranslations<E, T>> extends InheritedWidget {
+class InheritedLocaleData<E extends BaseAppLocale<E, T>, T extends BaseTranslations<E, T>> extends InheritedWidget {
```
Comment (Tienisto) [inline slang_flutter/lib/slang_flutter.dart:160]: While, I also like this more, `dart format` gives a different result

Comment (fzyzcjy) [reply slang_flutter/lib/slang_flutter.dart:160]: Seems my IDE uses `dart format --line-length 120`. Shall I `dart format` to modify it back?

Comment (Tienisto) [reply slang_flutter/lib/slang_flutter.dart:160]: Yes, you can do that :)

Comment (fzyzcjy) [reply slang_flutter/lib/slang_flutter.dart]: done

**Comment by Tienisto** [comment]
Nice!

**Pull request merged**